src/components/transactions/TxDetails/TxData/DecodedData/Multisend/index.tsx

```diff
@@ -51,6 +51,21 @@ export const Multisend = ({
 }: MultisendProps): ReactElement | null => {
   const [open, setOpen] = useState<Record<number, boolean>>()
 
+  // multiSend method receives one parameter `transactions`
+  const multiSendTransactions = txData?.dataDecoded?.parameters?.[0].valueDecoded
+
+  useEffect(() => {
+    // Initialise whether each transaction should be expanded or not
+    const multiSendTransactions = txData?.dataDecoded?.parameters?.[0]?.valueDecoded
+    if (!open && multiSendTransactions) {
+      setOpen(
+        multiSendTransactions.map(({ operation }) => {
+          return showDelegateCallWarning ? operation === Operation.DELEGATE : false
+        }),
+      )
+    }
+  }, [open, showDelegateCallWarning, txData?.dataDecoded?.parameters])
```

Comment:
> }, [open

Wouldn't this hook be triggered every time this flips?

Reply:
No, `open` is initially `undefined`. Would renaming it to `openMap` or something similar make more sense?